Source/common/SNTFileInfo.h

```diff
@@ -74,6 +74,12 @@
 - (NSString *)SHA256;
 
 ///
+///  Computes SHA1 and SHA256, caching the results.  This is slightly faster than computing each
+//   hash separately if you need them both.
+///
+- (void)precomputeSecureHashes;
```

Comment:
It makes more sense to me to keep the cached values with the SNTFileInfo object.  Now that the fileInfo object is decoupled from the SNTCommandFileInfo class, I'm not sure where the caller would cache SHA values for a particular file, other than using objc_setAssociatedObject again or keeping a dictionary mapping fileInfo objects to SHAs.  I could just take out the caching?